packages/shared/src/components/ExtensionOnboarding.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import Link from 'next/link';
+import Logo, { LogoPosition } from './Logo';
+import { OnboardingTitleGradient } from './onboarding/common';
+import { webappUrl } from '../lib/constants';
+import { Button, ButtonSize } from './buttons/Button';
+import { cloudinary } from '../lib/image';
+
+const ExtensionOnboarding = (): ReactElement => {
+  return (
+    <div className="flex overflow-hidden flex-col justify-center items-center px-7 antialiased text-center min-h-[100vh] max-h-[100vh]">
+      <Logo position={LogoPosition.Relative} logoClassName="h-logo-big" />
+
+      <OnboardingTitleGradient className="my-6 typo-mega2 tablet:typo-mega1">
+        Let&apos;s jump back&nbsp;in!
+      </OnboardingTitleGradient>
+
+      <p className="mb-9 tablet:mb-16 max-w-[25rem] typo-title3 tablet:typo-title2">
+        Please resume onboarding to unlock the entire feature suite of
+        daily.dev.
+        <br />
+        The magic awaits inside! ✨
+      </p>
+
+      <Link href={webappUrl}>
```

Comment:
I think we can go directly to `/onboarding` here.

Reply:
Forgot that one. Created a global constant and replaced the other instance of linking to onboarding page as well.